Macros/e3 Includes/e3_Classes_Bard.inc

```diff
@@ -114,7 +114,7 @@ SUB pauseTwist
 /if (${Debug} || ${Debug_Casting}) /echo |- pauseTwist ==>
 	/varset	returnTwist TRUE
 	/squelch /twist stop
-	/delay 20 !${Me.Casting.ID}
+	/delay 2 !${Me.Casting.ID}
```

Comment:
Why does this need to be changed from 2 seconds to .2 seconds? It's called in casting, loot, and buffcheck, so there might be side effects of this change.